app/src/main/java/com/orange/ouds/app/ui/components/radiobutton/RadioButtonItemDemoScreen.kt

```diff
@@ -0,0 +1,131 @@
+import com.orange.ouds.core.component.OudsRadioButtonItem
+import com.orange.ouds.core.theme.OudsTheme
+import com.orange.ouds.core.utilities.OudsPreview
+import com.orange.ouds.foundation.utilities.UiModePreviews
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun RadioButtonItemDemoScreen() = DemoScreen(rememberRadioButtonItemDemoState()) {
+    CustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+            ControlItemIconCustomization()
+            ControlItemDividerCustomization()
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_radioButton_radioButtonItem_outlined_label),
+                checked = outlined,
+                onCheckedChange = { outlined = it },
+            )
+            ControlItemInvertedCustomization()
+            ControlItemEnabledCustomization()
+            ControlItemReadOnlyCustomization()
+            ControlItemErrorCustomization()
+            ControlItemTextCustomization()
+            CustomizationTextField(
+                label = stringResource(R.string.app_components_radioButton_radioButtonItem_additionalText_label),
+                value = additionalText.orEmpty(),
+                onValueChange = { value -> additionalText = value }
+            )
+            ControlItemHelperTextCustomization()
+        }
+    ) {
+        LightDarkDemo {
+            RadioButtonItemDemo(state = this@DemoScreen)
+        }
+
+        RadioButtonItemDemoCodeSnippet(
+            state = this@DemoScreen,
+            modifier = Modifier
+                .padding(all = OudsTheme.spaces.fixed.medium)
+                .padding(top = OudsTheme.spaces.fixed.medium)
+        )
+    }
+}
+
+@Composable
+private fun RadioButtonItemDemo(state: RadioButtonItemDemoState) {
+    with(state) {
+        Column(modifier = Modifier.selectableGroup()) {
+            radioButtonItemDemoValues.forEach { radioButtonValue ->
+                OudsRadioButtonItem(
+                    selected = radioButtonValue == selectedValue,
+                    onClick = { selectedValue = radioButtonValue },
+                    text = text,
+                    additionalText = additionalText,
+                    helperText = helperText,
+                    icon = if (icon) OudsControlItem.Icon(painterResource(id = R.drawable.ic_heart)) else null,
+                    divider = divider,
+                    outlined = outlined,
+                    inverted = inverted,
+                    enabled = enabled,
+                    readOnly = readOnly,
+                    error = error
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun RadioButtonItemDemoCodeSnippet(state: RadioButtonItemDemoState, modifier: Modifier = Modifier) {
+    CodeSnippet(modifier = modifier) {
+        with(state) {
+            functionCall("OudsRadioButtonItem") {
```

Comment:
Following the radio button review, we should add `comment("First radio button")` here.